__tests__/app.test.js

```diff
@@ -62,6 +81,49 @@ describe('GET /api/articles', () => {
         })
       })
   })
+
+  test('Responds with the articles data sorted by the query', () => {
+    return request(app)
+    .get("/api/articles?sort_by=author")
+
+    .expect(200)
+    .then(({body}) => {
+      const {articles} = body
+
+      expect(articles).toBeSortedBy("author", {descending: true})
+    })
+  })
+
+  test('Responds with the articles data sorted descending or ascending', () => {
+    return request(app)
+    .get("/api/articles?order=desc")
+
+    .expect(200)
+    .then(({body}) => {
+      const {articles} = body
+
+      expect(articles).toBeSortedBy("created_at", {descending: true})
+    })
+  })
+
+  test('Responds with a 400 error when passed an invalid query', () => {
+    return request(app)
+    .get("/api/articles?sort_by=invalidColumn")
+    .expect(400)
+    .then(({body}) => {
+      expect(body.msg).toBe('Invalid input');
+    });
+  })
+
+  test('Responds with a 400 error when passed an invalid query', () => {
+    return request(app)
+    .get("/api/articles?sort_by=votes")
+
+    .expect(400)
+    .then(({body}) => {
+      expect(body.msg).toBe('Invalid input');
+    });
```

Comment:
I would argue both that 
- this test is the same concept as the test before it
- and being able to sort by votes sounds like it would be a useful feature rather than an error